### PR TITLE
{bio}[intel/2018a] Seaborn v0.9.0 w/ Python 2.7.14 + 3.6.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/62b5289eea6c69a8a52b7b073a427c2b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.11
See https://gist.github.com/5338bae5c40e34d438e6fee01b489a99 for a full test report.

File: easybuild/easyconfigs/s/Seaborn/Seaborn-0.9.0-intel-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Seaborn/Seaborn-0.9.0-intel-2018a-Python-2.7.14.eb]: @boegel please restore the original credit.

**Comment by boegel** [comment]
@verdurin Good to go now?

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**